src/components/editor/InputListLOC.jsx

```diff
@@ -50,13 +50,22 @@ class InputListLOC extends Component {
   }
 
   render() {
-    let lookupUri, isMandatory, isRepeatable
+    let isMandatory, isRepeatable
+
+    if (this.state.lookupUri === undefined) {
+      console.error("lookup URI was not defined!")
```

Comment:
If the lookup URI is undefined, what is the ultimate effect? Is it rendering the the `Typeahead` component with an `onFocus` value of `false`? Could/should we surface that error condition earlier rather than just logging an error?

Reply:
You would just see this:

<img width="460" alt="Screen Shot 2019-05-29 at 8 27 55 AM" src="https://user-images.githubusercontent.com/3093850/58569848-b70eaa00-81eb-11e9-8cab-62190a9ed2a0.png">